pdf/pdfdocument.cpp

```diff
@@ -119,9 +125,18 @@ void PDFDocument::setSource(const QString& source)
     }
 }
 
+void PDFDocument::requestUnLock(const QString& password)
+{
+  if (!isLocked())
+      return;
+
+  UnLockDocumentJob* job = new UnLockDocumentJob{ password };
```

Comment:
Ouch, that's extremely confusing IMO, not sure what the author was trying to achieve here, since all of these objects have defined constructors, anyway let's stick to the common usage and use simple braces like in other Qt code, also let's avoid no meaningful whitespaces around the braces, for example: d->thread->queueJob(job), also see: http://qt-project.org/wiki/Qt_Coding_Style#1d8ea7b3bfc87950e27892e5eac9c6f0